### ISSUE TITLE
Failed to create new bib file when exporting to a different disk partition

### ISSUE THREAD

**Issue by wohenbushuang** [retorquere/zotero-better-bibtex#1832]
**Support log ID:** URVZ7PCL-apse

**Exporter used:**

**Expected behavior:** 

I installed zotero in `C:`, and I want to export bib file to `S:\myLib_zotero.bib`, then 

**Actual behavior:** 

It cannot create the new bib file, and stucked. No error occured if I export bib file to `C:` partition.

support log:
> (3)(+0000000): {better-bibtex Better BibLaTeX error (worker 2)} +0 worker: <Error: Will not make relative S: makeDirs@resource://zotero-better-bibtex/worker/zotero.js?version=5.0.96.2&platform=win&translator=Better%20BibLaTeX&output=S%3A%5CmyLib_zotero.bib&localeDateOrder=ymd&debugEnabled=true&worker=2:44940:13 init@resource://zotero-better-bibtex/worker/zotero.js?version=5.0.96.2&platform=win&translator=Better%20BibLaTeX&output=S%3A%5CmyLib_zotero.bib&localeDateOrder=ymd&debugEnabled=true&worker=2:45010:9 Zotero__workerContext</ctx.onmessage@resource://zotero-better-bibtex/worker/zotero.js?version=5.0.96.2&platform=win&translator=Better%20BibLaTeX&output=S%3A%5CmyLib_zotero.bib&localeDateOrder=ymd&debugEnabled=true&worker=2:45086:9 >

**Comment by retorquere** [comment]
Can you get me a new support log from build 1028? It will fail again, but I need more information on the failure.

**Comment by retorquere** [comment]
I am looking to put out a new release and I'd prefer to have this fix in it, but I don't have a windows machine, so I really need the feedback.

**Comment by retorquere** [comment]
I really need that feedback

**Comment by retorquere** [comment]
Dude this is likely a 10 minute fix if you could just give me feedback.

**Comment by wohenbushuang** [comment]
@retorquere Sorry for late response, I don't browse github very often. New support log ID is `AYU26I5P-apse`
```
(3)(+0000003): {better-bibtex Better BibLaTeX (worker 1)} +0 makeDirs: {"context":{"exportFileData":false,"workerContext":{"debugEnabled":true,"localeDateOrder":"ymd","output":"S:\\aaa.bib","platform":"win","translator":"Better BibLaTeX","version":"5.0.96.2","worker":"1"}},"path":"S:"} 
```

**Comment by retorquere** [comment]
Don't you get email notifications?

The problem may already be fixed on 5.4.23, please try that first..

**Comment by wohenbushuang** [comment]
I have updated to 5.4.23 and **confirmed the problem have been solved**, thanks a lot~

I don't check my email very often, either, actually haven't since this issue was created. Instant messaging is more popular in some countries. Sorry for any inconvenience.
